### PR TITLE
refactor(describe): make LB/backend etc. service describers agnostic to env

### PR DESCRIPTION
**Pull request by iamhopaul123**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: internal/pkg/describe/rd_web_service.go [hunk 1]
```diff
@@ -55,6 +54,15 @@ func NewRDWebServiceDescriber(opt NewServiceConfig) (*RDWebServiceDescriber, err
 	return describer, nil
 }
 
+// ServiceARN retrieves the ARN of the app runner service.
+func (d *RDWebServiceDescriber) ServiceARN(env string) (string, error) {
+	err := d.initClients(env)
+	if err != nil {
+		return "", err
+	}
+	return d.envSvcDescribers[env].ServiceARN()
```
Comment (dannyrandall) [inline internal/pkg/describe/rd_web_service.go]: probably should check and make sure their is a describer for `env` before calling the function so we don't panic
```
ard, ok := d.envSvcDescriber[env]
if !ok || ard == nil {
    // error
}

return ard.ServiceARN()
```

File: internal/pkg/describe/service.go [hunk 2]
```diff
@@ -86,74 +86,111 @@ type serviceStackDescriber struct {
...
+	ignoredResources := map[string]bool{
+		rulePriorityFunction: true,
+		waitCondition:        true,
+		waitConditionHandle:  true,
+	}
+	for _, svcResource := range svcResources {
+		if ignoredResources[svcResource.Type] {
+			continue
+		}
+		resources = append(resources, svcResource)
```
Comment (dannyrandall) [inline internal/pkg/describe/service.go]: nit
```suggestion
		if !ignoredResources[svcResource.Type] {
			resources = append(resources, svcResource)
		}
```

File: internal/pkg/describe/rd_web_service.go (hunk 1 above)
Comment (iamhopaul123) [reply internal/pkg/describe/rd_web_service.go]: Good call! Addressed

**Review by dannyrandall — APPROVED** [review]
lgtm🥇

**Review by huanjani — APPROVED** [review]
🚢

**Pull request merged**